src/app.coffee

```diff
@@ -36,15 +39,21 @@ defaults =
   driver: "memory"
   params: {}
   address: null,
-  
-if fs.existsSync(CONFIG_FILE)
-  try
-    config = JSON.parse(fs.readFileSync(CONFIG_FILE))
-  catch err
-    console.error err
-    process.exit 1
-else
-  config = {}
+  port: null
```

Comment:
I'm assuming this is left over from rebasing on top of me doing the same change? If so this should probably be removed and the commit message amended

Reply:
the `port: null`? is for keep the available options in `defaults = {}`

Reply:
Ah, gotcha. Can you make that a separate commit so it's clearer what's going on? Ping me in IRC if you want help